plugins/services/src/js/containers/services/__tests__/ServicesTable-test.js

```diff
@@ -95,6 +96,15 @@ describe("ServicesTable", function() {
       expect(ReactDOM.findDOMNode(cpusCell).textContent).toEqual("1");
     });
 
+    it("renders resources/stats gpus property", function() {
+      var gpusCell = ReactDOM.render(
```

Comment:
It seems like I overlooked this part while trying to move to enzyme, would you mind moving these tests to use enzyme? A no would also be fine, I would file a jira issue then

Reply:
I would like to see this moved to enzyme too